web/packages/teleport/src/Login/Login.tsx

```diff
@@ -136,7 +136,7 @@ function LicenseAcknowledgement({
             <>
               By clicking continue, you agree to our{' '}
               <Link
-                href="https://github.com/gravitational/teleport/blob/master/LICENSE-community"
+                href="https://github.com/gravitational/teleport/blob/master/build.assets/LICENSE-community"
```

Comment:
This link in the web UI will be broken in old releases. I'm not sure it's worth trying to avoid, as we'd have to keep the license in both places for some period of time.

(We really should consider having some sort of link shortener for any links we use in-product)